api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -105,6 +105,8 @@ message RBAC {
   Action action = 1 [(validate.rules).enum = {defined_only: true}];
 
   // Maps from policy name to policy. A match occurs when at least one policy matches the request.
+  // The policies are evaluated in the order of the policy name. The policy with the smallest name
```

Comment:
Just a drive-by, but I think we should be clear about what "smallest" means. Should it say lexicographic order? e.g. "The policies are evaluated in lexicographic order."

Reply:
thanks for the comment, updated.